lib/bait/ast/table.bt

```diff
@@ -3,7 +3,8 @@
 package ast
 
 pub struct Table {
-	pub mut global_scope Scope
+	pub mut global_scope &Scope // TODO entirely replace by scopes
```

Comment:
This field could perfectly function as an entry point to all other areas. As a mere suggestion, you could have the primitive types and builtin functions (if there are any) be added to this scope.

Reply:
Another good find! 
Actually I have already in mind to add a `universe` scope or reintroduce the `global` scope for this purpose.
But primitive types are still resolved differently, so this is for a follow-up change.

Reply:
It's a good idea to use `universe` as the root scope ;)

Reply:
Cannot be confused with `global` keyword

Reply:
I suggest not using `global`, use static package-level variables instead.